config/rubocop.yml

```diff
@@ -92,3 +92,8 @@ AllCops:
 Style/BlockDelimiters:
   Exclude:
     - "spec/**/*"
+
+# Ruby version
+
+AllCops:
+  TargetRubyVersion: 2.3
```

Comment:
`TargetRubyVersion: ~` imho is better as tries to use .ruby-version, making it more flexible